lib/ohai/plugins/kernel.rb

```diff
@@ -116,7 +116,12 @@ def os_lookup(sys_type)
     so = shell_out("env lsmod")
     so.stdout.lines do |line|
       if line =~ /([a-zA-Z0-9\_]+)\s+(\d+)\s+(\d+)/
+        _status, version, _stderr = run_command(
```

Comment:
OK, I did not know about `run_command`. 

Regarding the `"modinfo #{mod} -F version"` I am not confortable with this command because it only reflects the module present on the file system not the version loaded. I will figured out a way to do it.